### DISCUSSION TITLE
Use cross-env for dev script

### DISCUSSION THREAD

**Discussion by kleenkanteen** [lukevella/rallly#756] [General]
In `apps\landing\package.json`, the line `"dev": "TAILWIND_MODE=watch next dev",` errors for me on windows 10 when I do `yarn dev`. This because windows doesn't recognize that format. I installed the `cross-env` package and changed the line to `"dev": "cross-env TAILWIND_MODE=watch next dev",`, and the error when running it went away. Ok to make a PR for it?

**Comment by lukevella** [comment]
Hi @kleenkanteen, sure that sounds fine to me. Feel free to open a PR adding `cross-env` to the scripts that need it.
